src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -121,3 +121,85 @@ end
 
 
 
+
+"""
+    struct ConstantLifetimeChargeTrappingModel{T <: SSDFloat} <: AbstractChargeTrappingModel{T}
+        
+This constant-lifetime-based charge trapping model is similar to the Boggs model, which is constant-mean-free-path based.
+
+The model implemented has two sets of parameters for the sensitive (bulk) and inactive (dead layer/surface layer) volume respectively.
+The inactive layer effect will only be considered if the corresponding `virtual_drift_volume` is defined in the configuration file.
```

Comment:
Is it a `virtual_drift_volume`? You call it `inactive_layer_geometry` in the documentation.